src/main/java/database/service/entity/Student.java

```diff
@@ -0,0 +1,57 @@
+package database.service.entity;
+
+import java.util.Objects;
+
+public class Student extends BaseEntity {
+    private String fullName;
+    private Integer courseId;
```

Comment:
Let's assume that student can attend to multiple courses. Please change it to collection.